### PR TITLE
compstudio.in

### PR DESCRIPTION
**Pull request by scripthunter7**
https://github.com/AdguardTeam/AdguardFilters/issues/127746

Ads also removed.

Since I don't always understand where to put a rule in the file, I hope I didn't mess it up 🙂

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -667,12 +667,12 @@ fcportables.com,ps2-bios.com##div[id$="____equal"]:not(.ad)
...
...tiq.net,gadgets.techymedies.com,arabgamingworld.com,aktualnosci.news,frontedelblog.it,nachrichten.cyou,souq-design.com,inputmag.dk,comoapple.com,thepicpedia.com,deluxe.news,ta1lsx.com,dailybanner.co.uk,codelist.biz,intozoom.com,builder.id,aloomag.ir,nepaleseteacher.org,veganab.co,tmz.ng,newzpanda.com,yossr.com,how2electronics.com,poradniki.net,expatguideturkey.com,sorumatik.co,databaseitalia.it,zodhacks.com,dnaofsports.com,bollywoodhindi.in,nubng.com,vga4a.com,thetimesweekly.com,kdramasurdu.net,raqmedia.com,newsauto.it,compuhoy.com,gatexplore.com,rolasdanet.com,avmkatalog.com,aktuelsanat.net,techhelpbd.com,ozkanalkan.net,apprepack.com,legrandreveil.co,alghad.com,scriptciyiz.org,oicanadian.com,ifon.ca,cnwintech.com,washingtoninformer.com,thetruedefender.com,muzikonair.com,web4free.in,numberozo.com,haronefit.com,tutocad.com,omerbozdi.com.tr,softvela.us,wallpapers.ispazio.net,bsaktuell.de,koinbox.net,inventiva.co.in,estafed1.com,tntmac.com,latestreview.tech,curiosandosimpara.com,vodnews.pl,tipslearn.com,rebe1scum.com,animedevil.com,fullversionforever.com,stockrom.net,thenextdroid.com,jogging-plus.com,cozumpark.com,rontechtips.com,trivela.com.br,cimaroom.com,compstudio.in#@#.sidebar-ad
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:675]: Please move the domain to the beginning of the rules.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -1151,6 +1151,12 @@ collive.com##.premium-post-ad
 m.xcum.com##.mob_under_player
 roadtovr.com##footer > div.td-post-content
 nubng.com,techhelpbd.com###stream-item-widget-5
+compstudio.in##.stream-item-top-wrapper
+compstudio.in##.stream-item-above-post
+compstudio.in##.stream-item-above-post-content
+compstudio.in##.stream-item-widget
+compstudio.in##.stream-item-inline-post
+compstudio.in##.stream-item-below-post-comments
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:1159]: I'd say you can use `compstudio.in##div[class^="stream-item"]`, but please check this (it's ok on my end).

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -1937,6 +1937,8 @@ imgrum.org#?#.module_blog_page > .blogpost_user_meta[-ext-has=">.adsbygoogle"]
 cutestat.com#?#.stat_details > div.marginBottom_10 > div.stat_half_left[-ext-has="> div > script:contains(window._mNHandle.queue.push)"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/6638
 rarbg2020.org,rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unblocked.lol,rarbg.pbproxy2.co#?#td[align="justify"] > div > table[width="100%"][class="lista"] > tbody > tr:has(tr > td + td[style*="height:"])
+! https://github.com/AdguardTeam/AdguardFilters/issues/127746
+compstudio.in#$#.adtester-container { position: absolute !important; top: -9999px !important; }
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:1941]: Where do you need the rule? The elements are already blocked.

Comment (scripthunter7) [reply EnglishFilter/sections/css_extended.txt:1941]: It removes blank spaces between paragraphs (these do not always appear):

image

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (scripthunter7) [reply EnglishFilter/sections/specific.txt:1159]: I thought about what you said. I thought this version was more specific. OK, I'll change it to attribute selector

File: EnglishFilter/sections/css_extended.txt (hunk 3 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:1941]: You know, it's hidden on my end by some generic rules, but maybe this is a rare case.
Please move this rule with task comment to `general_extensions` file.

Comment (scripthunter7) [reply EnglishFilter/sections/css_extended.txt]: I did it, I hope everything is fine

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 3 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: Yep, thank you;)

**Pull request merged**